.gitignore

```diff
@@ -15,3 +15,5 @@ export_presets.cfg
 .vs
 # Mac-specific ignores
 .DS_Store
+*.import
+*.import
```

Comment:
don't add the same entry again. I have them higher up in the Godot section.

Reply:
Alright, I got rid of the duplicates as a new change.